lib/launcher.rb

```diff
@@ -68,7 +68,7 @@ def https_app(https_port, plugins = https_plugins)
 
       ssl_options = OpenSSL::SSL::SSLContext::DEFAULT_PARAMS[:options]
       ssl_options |= OpenSSL::SSL::OP_CIPHER_SERVER_PREFERENCE if defined?(OpenSSL::SSL::OP_CIPHER_SERVER_PREFERENCE)
-      # This is required to disable SSLv3 on Ruby 1.8.7
+      # hardwire disabling of specific SSL/TLS versions
```

Comment:
This is mainly depending onto libssl, so we should keep this.

Reply:
Can you please drop this line from the patch? I vastly refactored launcher.rb in my other PR, there are multiple commits in a single PR to keep authorship, this got rewritten too. It will only cause conflicts for me.